### DISCUSSION TITLE
ThinStation 7.0

### DISCUSSION THREAD

**Discussion by Doncuppjr** [Thinstation/thinstation#863] [Announcements]
# Release Announcement: ThinStation 7.0

We are thrilled to announce the release of **ThinStation 7.0**, a significant update that marks a new era for our lightweight thin client solution. This release brings numerous changes aimed at modernizing the platform and streamlining its development and maintenance. Here's what’s new:

### Rebased to DNF Binaries
ThinStation has been rebased to use DNF binaries, leveraging the robust package management system of Fedora. This change enhances compatibility and aligns ThinStation with a modern, actively maintained ecosystem.

### Core Package Migration
Core packages have been migrated to the new base, replacing legacy custom packages. This provides a cleaner and more consistent foundation, simplifying updates and maintenance.

### Goodbye Custom Binaries
ThinStation no longer relies on custom binaries. This shift ensures better integration with upstream packages and simplifies troubleshooting and future enhancements.

### PipeWire Integration
We've transitioned from PulseAudio to **PipeWire**, a next-generation multimedia framework. PipeWire delivers lower latency, improved performance, and better support for modern audio and video use cases.

### Configuration Updates
Configurations are undergoing significant changes. While the base system is solid, we recognize that existing setups might need some adjustments to work seamlessly. Your feedback during this transitional phase is invaluable.

---

## What’s Next?
With this release, we're focusing on making ThinStation more accessible, reliable, and adaptable for current and future deployments. We're also working to improve documentation and configuration tools to ensure a smoother user experience.

---

## Call for Community Contributions
This release is a fresh start, and we invite the community to contribute! Whether it's testing configurations, submitting patches, or sharing use cases, your involvement will help shape the future of ThinStation.

---

Thank you for your support and patience as we continue to improve ThinStation. We’re excited about the possibilities that ThinStation 7.0 opens up, and we look forward to hearing your feedback.

**Get started with ThinStation 7.0 today!**
https://www.thinstation.org/TS-7.0.0-Installer-1209.iso

**Comment by VolksDude** [comment]
Awesome! I'll set it up and let you know if I find issues...

**Comment by fanuelsen** [comment]
Your comitment to Thinstation is truly amazing @Doncuppjr

**Reply by Doncuppjr** [reply]
I think it's a useful project. It's been a great teacher.
